Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "dusk-hades"
-version = "0.13.0"
+version = "0.14.0"
```

Comment:
This should be probably `0.13.1` since the public API didn't change, right?

Reply:
As discussed, if any breaking change is performed on a public signature or trait, we should increment as breaking.

The gadget strategy exposes the variables of plonk

Reply:
As discussed on the stand up, let's following this:

1. If the public API does not change is not a breaking change
2. If the public API changes, is a breaking change

In this case the public API changes because the dependency types (plonk's type) are part of structure / signature of Hades, therefore is a breaking change; then the version should be `0.14.0`